README.md

```diff
@@ -343,31 +343,6 @@ All dbt commands are supported.
 
 ### Custom configurations
 
-#### General

```

Comment:
![image](https://github.com/user-attachments/assets/a3812b3b-ea27-4668-9c66-4cf9440da2e0)
This is not related to QVCI. Why are we mentioning about 1.8.x and again 1.8.2 in python compatibility and in dbt dependent package table, specified as1.8.x is compatible with 1.2.0 and 1.30 of dbt-teradata-util. Is this correct?

Reply:
At a minimum, you need to specify `host`, `user`, `password`, `schema` (database), `tmode`.

tmode is default to ANSI in https://github.com/Teradata/dbt-teradata/blob/3d42c353dc1e3fa3a08e64720a051ebd678149b6/dbt/adapters/teradata/connections.py#L28C4-L28C34 and its not mandatory. We can remove tmode from At a minimum section

Reply:
Yes, 1.8.x is compatible with 1.2.0 and 1.30 of dbt-teradata-utils